package.json

```
@@ -5,11 +5,11 @@
   "author": "Sentry",
   "dependencies": {
     "@types/react": ">=16",
-    "@types/react-dom": ">=16",
+    "@types/react-dom": "^17.0.0",
```

we can't actually do `^` here since it would force anyone using this package to be on 17.
```suggestion
    "@types/react-dom": ">=17.0.0",
```